README.dev.md

```diff
@@ -38,63 +67,120 @@ such as Mac via Virtualbox (but everything will be 2-3x slower).
   LXC installation instructions][vagrant-lxc] to get Vagrant with LXC
   for your platform.
 
-* If your host is OS X or older Linux, [download VirtualBox][vbox-dl],
+* If your host is older Linux, [download VirtualBox][vbox-dl],
   [download Vagrant][vagrant-dl], and install them both.
+  
+* For all Linux platforms: skip to the universal instructions below
+  OS X instructions and Windows instructions. 
 
-* If you're on OS X and have VMWare, it should be possible to patch
-  Vagrantfile to use the VMWare vagrant provider which should perform
-  much better than Virtualbox.  Patches to do this by default if
-  VMWare is available are welcome!
+The following bullet is for starting the setup of the environment on OS X:
 
-* On Windows: You can use Vagrant and Virtualbox/VMWare on Windows
-  with Cygwin, similar to the Mac setup.  Be sure to create your git
-  clone using `git clone https://github.com/zulip/zulip.git -c
-  core.autocrlf=false` to avoid Windows line endings being added to
-  files (this causes weird errors).
+* [download VirtualBox][vbox-dl], [download Vagrant][vagrant-dl], and
+  install them both.
+
+  Note: If you're on OS X and have VMWare, it should be possible to patch
+  Vagrantfile to use the VMWare vagrant provider, which should perform
+  much better than Virtualbox. Patches to do this by default are welcome!
+
+The following bullet is for starting the setup of the environment on Windows:
+
+* [Download VirtualBox][vbox-dl], [download Vagrant][vagrant-dl],
+  [download Cygwin][cygwin-dl], and install all three.
```

Comment:
I feel like this section could also use more eye-catching labels for the platforms and/or links to the relevant sections.